### ISSUE TITLE
Curse Launcher Issue

### ISSUE THREAD

**Issue by steamboat28** [GrowthcraftCE/Growthcraft-1.7#190]
I searched for this so I wouldn't clutter the issue tracker, but I couldn't find anything like this already spoken about.

I was very excited to see this CE, and installed it into a (mostly) blank profile on the Curse launcher. No items, blocks, or mechanics were added. None of the Growthcraft stuff was visible in NEI or in the inventory tabs. I uninstalled and tried again, with the same results.

The issue was resolved by downloading from the link in the MinecraftForum.net post (which took me to a Curse download page in my browser), but anytime I attempt to install from the Curse Voice Client, I get nothing. Version mismatch, maybe?

**Comment by IceDragon200** [comment]
@steamboat28 Did you check the modlist in minecraft for Growthcraft?
Also you could toggle the `growthcraft/cellar.cfg`'s `debug` option, which would print some info to the console (that way you know its working).
I know next to nothing about the Curse Launcher, so I can't help you there.

Also ALL the versions for each module are the same, regardless of which module we updated
so `growthcraft-2.4.0` will ship with everything as 2.4.0 (API included)

**Comment by IceDragon200** [comment]
@steamboat28 PLING, Any news on this issue, I'd like to start cleaning up issues so I can focus on the important ones

**Comment by steamboat28** [comment]
Sorry it took me so long to get back to this.

I started a brand new instance on the Curse launcher; a blank 1.7.10 install with the latest stable Forge release installed. I then added GCE from the Curse launcher and started a creative world.

There was no creative tab for GCE, nor even a config folder for the mod. Upon looking in the mods folder, I discovered this might (emphasis here; i don't know for certain) be part of the issue. I then added NEI and its dependency, and I unzipped the bundle (without removing it from the mods folder) and tried again, whereupon I was greeted with [this crash report](http://pastebin.com/MRbxeVqu). Trying again with NEI and CodeChickenCore disabled got me a similar crash report. Removing the .zip file of the GCE bundle and leaving the GCE module .jars themselves made no difference; [yet another crash report](http://pastebin.com/3VcDLgA7).
